cmd/fossa/update.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"github.com/blang/semver"
+	logging "github.com/op/go-logging"
+	"github.com/rhysd/go-github-selfupdate/selfupdate"
+	"github.com/urfave/cli"
+)
+
+var updateLogger = logging.MustGetLogger("update")
+var updateEndpoint := "fossas/fossa-cli"
+
+func checkUpdate() error {
```

Comment:
This should return `(bool, error)`. Don't use `error` to represent things that are not errors.